tools/gulp/docs/generateReactExample.js

```diff
@@ -0,0 +1,97 @@
+const MemoryFS = require('memory-fs');
+const path = require('path');
+const savePage = require('./savePage');
+const webpack = require('webpack');
+
+/**
+ * Creates an HTML page using the React component and no additional UI.
+ * This can then be viewed in a browser, or rendered in an iFrame in
+ * the documentation.
+ * @param {Object} page
+ * @param {String} rootPath - Root docs site path
+ * @return {Promise}
+ */
+function generateReactExample(page, rootPath) {
```

Comment:
ℹ️ We use Webpack to bundle and compile each React example, then place the output as an inline `script` at the bottom of the generated HTML page for that example.